### ISSUE TITLE
Enable subtitles?

### ISSUE THREAD

**Issue by joelcald** [ca0abinary/plugin.video.jwtv-unofficial#4]
Is it possible to enable subtitles? This is very helpful for language learners. Thanks for the great plugin!

![image](https://cloud.githubusercontent.com/assets/10685591/8917249/f3c7a23a-347e-11e5-8e97-ab03fba06158.png)

**Comment by ca0abinary** [comment]
@joelcald Added to master, could you test it? I can't seem to find any videos with subtitles anymore. :disappointed:

**Comment by joelcald** [comment]
Tested it on Kodi for Android and it seems to work! I had to manually install the zip since it didn't update to 1.8.3 automatically and then fully exit and restart Kodi. Also, if you select **French** as the language, the four main menu items are missing. Everything worked in **Spanish** and **English**.

To test the subtitles, I used feature length films (*The Prodigal Returns, These Words...*) and some of the original music videos in any language other than English.

I'll test it on OSMC tonight. Thanks!
